File: src/DashboardUI/src/data-contracts/SiteDigest.ts
import WaterRightDigest from './WaterRightsDigest';

interface SiteDigest {
  siteUuid: string;
  siteNativeId: string;
  siteType: string;
  siteName: string;
  hasTimeSeriesData: boolean;
  timeSeriesVariableTypes: string[];
  waterRightsDigests: WaterRightDigest[];
}

export default SiteDigest;

File: src/API/WesternStatesWater.WestDaat.Common/DataContracts/SiteDigest.cs
namespace WesternStatesWater.WestDaat.Common.DataContracts
{
    public class SiteDigest
    {
        public string SiteUuid { get; set; }
        public string SiteNativeId { get; set; }
        public string SiteName { get; set; }
        public string SiteType { get; set; }
        public bool HasTimeSeriesData { get; set; }
        public List<string> TimeSeriesVariableTypes { get; set; }
        public List<WaterRightsDigest> WaterRightsDigests { get; set; } = new List<WaterRightsDigest>();
    }
}